### PR TITLE
Sailfish

### PR DESCRIPTION
**Pull request by JensTimmerman**

### CODE REVIEW

File: easybuild/easyconfigs/s/Sailfish/Sailfish-0.6.3-foss-2014b.eb [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+ isoform abundances directly from a set of reference sequences and RNA-seq reads. """
+
+toolchain = {'name': 'foss', 'version': '2014b'}
+
+source_urls = ['https://github.com/kingsfordgroup/sailfish/archive/']
+sources = ['v%(version)s.tar.gz']
+
+python = 'Python'
+pyver = '2.7.8'
+versionsuffix = '-%s-%s' % (python, pyver)
```
Comment (JensTimmerman) [inline easybuild/easyconfigs/s/Sailfish/Sailfish-0.6.3-foss-2014b.eb]: you're right, this one was added by mistake

Comment (boegel) [inline easybuild/easyconfigs/s/Sailfish/Sailfish-0.6.3-foss-2014b.eb]: OK, so please remove it?

**Comment by JensTimmerman** [comment]
done

**Comment by boegel** [comment]
@JensTimmerman: needs more work...

**Comment by JensTimmerman** [comment]
jenkins test this please

**Comment by boegel** [comment]
Please sync whole branch with develop.

**Comment by JensTimmerman** [comment]
@boegel rebased on develop

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 7 out of 7 (7 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/6db434eaf2afb17d514a for a full test report.

**Comment by boegel** [comment]
Alrighty, going in, thanks @JensTimmerman!

**Pull request merged**